reactnative/RNPlayground/app/screens/AboutScreen.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from "react"
+import { observer } from "mobx-react-lite"
+import { StackScreenProps } from "@react-navigation/stack"
+import { AppStackScreenProps } from "../navigators"
+import { Image, ImageStyle, TextStyle, View, ViewStyle } from "react-native"
+import {
+  Text, Screen
+} from "../components"
+import { isRTL } from "../i18n"
+import { colors, spacing } from "../theme"
+import { useSafeAreaInsetsStyle } from "../utils/useSafeAreaInsetsStyle"
```

Comment:
what does the `useSafeAreaInsetStyle` exactly does?

Reply:
This is another import to support displaying elements of the navigators display correctly on devices with notches. Or, in other words, we're in iOS territory now have to account for top notches. 🤢 

More info here https://reactnavigation.org/docs/handling-safe-area/

Reply:
Is this only for iOS? Native Android has notion of insets too so I thought this was doing the same thing there too

Reply:
You're right, it doesn't just handle iOS's top notches. All OS elements are accounted for when defining a `SafeArea`, e.g. status bars, home indictaors, navigation bars.

https://github.com/infinitered/ignite/blob/master/docs/Utils-useSafeAreaInsetsStyle.md - Ignite has documented its use here